### PR TITLE
Fix import external addons action

### PR DESCRIPTION
**Pull request by emosbaugh**
#### What this PR does / why we need it:

Fix the action
Explicitly fetching external add-ons from s3Override rather than falling back on 404s

### CODE REVIEW

File: scripts/common/addon.sh [hunk 1]
```diff
@@ -62,7 +62,7 @@ function addon_fetch() {
     if [ "$AIRGAP" != "1" ]; then
         if [ -n "$s3Override" ]; then
             rm -rf $DIR/addons/$name/$version # Cleanup broken/incompatible addons from failed runs
-            addon_fetch_no_cache "$s3Override"
+            addon_fetch_cache "$name-$version.tar.gz" "$s3Override"
```
Comment (emosbaugh) [inline scripts/common/addon.sh:65]: is it ok to fetch s3Override with cache?

File: scripts/common/common.sh [hunk 2]
```diff
@@ -53,28 +41,31 @@ function package_download() {
...
+    checksum="$(grep -F "${package}" assets/Manifest | awk 'NR == 1 {print $3}')"
 
     if [ -n "${etag}" ] && ! package_matches_checksum "${package}" "${checksum}" ; then
         etag=
     fi
 
     local package_url=
-    package_url="$(get_dist_url)/${package}"
-
-    # if the package does not exist as a "normal" add-on, try the external add-on cache
```
Comment (emosbaugh) [inline scripts/common/common.sh:66]: this was awkward

File: scripts/common/addon.sh (hunk 1 above)
Comment (laverya) [reply scripts/common/addon.sh:65]: I think it's fine
we repull if the sha changed, right?

**Review by laverya — APPROVED** [review]

File: scripts/common/addon.sh (hunk 1 above)
Comment (emosbaugh) [reply scripts/common/addon.sh:65]: ya. looks like we check the etag in s3

**Pull request merged**